include/RAJA/reducers.hxx

```diff
@@ -59,7 +59,7 @@
 /// Define max number of reductions allowed within a RAJA traversal
 /// (sizes of shared memory blocks for reductions are set based on this value)
 ///
-#define RAJA_MAX_REDUCE_VARS (8)
+#define RAJA_MAX_REDUCE_VARS (24)
```

Comment:
This should also get changed back to what they had; Justis changed this when we was working on FFT2d. Potentially also add it as a CMake variable so it's easier to set at compile time?